### PR TITLE
Heroku Deployment of the App

### LINKED ISSUE #267: Deployment on Heroku
## Description
As a contributor,
I need deploy the systers.io app on Heroku

### Update [Required]
- [ ] A link with NodeJS app deployed

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
20 hours

@Janiceilene @bethanyr @ancyp @Rupinderwadali Can you please assign me the issue?

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
Deploying the app on Heroku.

Fixes #267

### Type of Change:

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation
  
**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
It has been tested locally and the app has been successfully deployed to https://murmuring-refuge-48696.herokuapp.com/.

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 
  
**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: src/app/common.service.ts [hunk 1]
```diff
@@ -12,15 +12,15 @@ export class CommonService {
   constructor(private http: Http) { }
 
   saveUser(user) {
-    return this.http.post('http://localhost:8080/api/SaveUser/', user)
+    return this.http.post('https://murmuring-refuge-48696.herokuapp.com/api/SaveUser/', user)
```
Comment (bethanyr) [inline src/app/common.service.ts]: Can you update to be the systems.io heroku url? And create an issue to not have this hard coded?

File: src/app/common.service.ts [hunk 2]
```diff
@@ -12,15 +12,15 @@ export class CommonService {
...
       .map((response: Response) => response.json());
   }
   GetUser() {
-    return this.http.get('http://localhost:8080/api/getUser/')
+    return this.http.get('https://murmuring-refuge-48696.herokuapp.com/api/getUser/')
```
Comment (bethanyr) [inline src/app/common.service.ts]: See previous comment about url

File: src/app/common.service.ts [hunk 3]
```diff
@@ -12,15 +12,15 @@ export class CommonService {
...
       .map((response: Response) => response.json());
   }
   deleteUser(id) {
-    return this.http.post('http://localhost:8080/api/deleteUser/', { 'id': id })
+    return this.http.post('https://murmuring-refuge-48696.herokuapp.com/api/deleteUser/', { 'id': id })
```
Comment (bethanyr) [inline src/app/common.service.ts]: See previous comment about url

File: src/app/common.service.ts (hunk 1 above)
Comment (BaaniLeen) [reply src/app/common.service.ts]: Sure. Done :)

**Comment by BaaniLeen** [comment]
Thanks @bethanyr ! :)
Made the changes. Once you and @ancyp review them, I can squash all the commits together :)

**Comment by bethanyr** [comment]
@ancyp - are you OK if we create issues for @BaaniLeen to fix the nav issue and populate the programs data so we can get this PR merged in?

**Pull request merged**